roles/bwc/tasks/ldap.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Setup st2.conf auth backend to LDAP
+  become: yes
+  ini_file:
+    dest: "{{ st2_config_file_path }}"
+    section: auth
+    option: backend
+    value: ldap
+    backup: yes
```

Comment:
Notify st2 `restart` or `reload` handler (if enough) trigger when changing the `st2.conf` file.
Example: https://github.com/StackStorm/ansible-st2/blob/master/roles/st2/tasks/auth.yml#L34